graylog2-server/src/main/java/org/graylog2/migrations/V20231107164300_CreateDataNodeManagerRole.java

```diff
@@ -38,6 +38,6 @@ public ZonedDateTime createdAt() {
     @Override
     public void upgrade() {
         helpers.ensureBuiltinRole("Data Node Manager", "Grants control to manage the data node cluster (built-in)",
-                Set.of(RestPermissions.DATANODE_REMOVE, RestPermissions.DATANODE_RESET));
+                Set.of(RestPermissions.DATANODE_REMOVE, RestPermissions.DATANODE_RESET, RestPermissions.DATANODE_STOP, RestPermissions.DATANODE_START));
```

Comment:
Is it safe to modify an existing migration? Between releases only developers will be possibly influenced by this change, but maybe it would be safer to create a completely new migration?

Reply:
AFAIK the migrations will always run unless you implement a mechanism, so this should be fine in between releases.
But if you think it would be better, I can separate that into a new migration.

Reply:
You are right, it will run every time. Then it's all good!